doc/lua_api.md

```diff
@@ -7791,20 +7698,15 @@ child will follow movement and rotation of that bone.
     * Sets the position of the object.
     * No-op if object is attached.
     * `pos` is a vector `{x=num, y=num, z=num}`
-* `add_pos(pos)`:
-    * Changes position by adding to the current position.
-    * No-op if object is attached.
-    * `pos` is a vector `{x=num, y=num, z=num}`.
-    * In comparison to using `set_pos`, `add_pos` will avoid synchronization problems.
 * `get_velocity()`: returns the velocity, a vector.
 * `add_velocity(vel)`
     * Changes velocity by adding to the current velocity.
     * `vel` is a vector, e.g. `{x=0.0, y=2.3, z=1.0}`
-    * In comparison to using `get_velocity`, adding the velocity and then using
-      `set_velocity`, `add_velocity` is supposed to avoid synchronization problems.
-      Additionally, players also do not support `set_velocity`.
+    * In comparison to using get_velocity, adding the velocity and then using
+      set_velocity, add_velocity is supposed to avoid synchronization problems.
+      Additionally, players also do not support set_velocity.
```

Comment:
I thought function names were supposed to be in code formatting?

Reply:
You are right, changed